docs/release-notes/8.8.asciidoc

```diff
@@ -15,16 +15,10 @@
 [[breaking-changes-8.8.2]]
 ==== Breaking changes
 
-//tag::breaking-changes[]
-// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
-// THIS ALSO MEANS IF YOU USE LINKS HERE, THEY SHOULD BE FULL URLS WITH NO ATTRIBUTES
-
 :pull: https://github.com/elastic/kibana/pull/
```

Comment:
We can probably also delete this line, since we no longer need to define the pull attribute in a snippet (it's already globally defined).

Reply:
```suggestion
```